docs/release_notes.rst

```diff
@@ -35,12 +35,12 @@ runs on most Windows®, Linux® and macOS® platforms.
 `main Connector
 repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 
-Version 1.1.0
+Version 1.1.1
 ~~~~~~~~~~~~~
 
-*RTI Connector* 1.1.0 is built on `RTI Connext DDS 6.1.0 <https://community.rti.com/documentation/rti-connext-dds-610>`__.
+*RTI Connector* 1.1.1 is built on `RTI Connext DDS 6.1.0 <https://community.rti.com/documentation/rti-connext-dds-610>`__.
```

Comment:
We need to check with Elaine what this should link to, I'm not sure if we have documentation for the patch on the community or not (if not this is fine as is)